src/app.tsx

```diff
@@ -34,28 +34,45 @@ import { NavigatorCardBody } from "./navigator-card-body.jsx";
 import { SidebarPanelDetails } from "./sidebar.jsx";
 import { NavigatorCardHeader } from "./header.jsx";
 import { usePageLocation } from "hooks.js";
-import { fsinfo } from "./fsinfo";
+import { fsinfo, FileInfo } from "./fsinfo";
 
 superuser.reload_page_on_change();
 
+type Alert = {
+    key: string,
+    title: string,
+    variant: AlertVariant,
+};
+
+type NavigatorFileInfo = {
```

Comment:
I guess that should rather `extends FileInfo` from `fsinfo.ts?`

Reply:
Also: in general, I think it makes more sense to use `interface` for this sort of thing...

Reply:
Right, but I gotta drop fields as well :)